Orange-Co/DDANZI_Server
# Feat: implement item APIs

create SignedUrl
- create signed url of google cloud storage
- test upload image using signed url

check product
- get most similar product to communicate ai server

confirm product
-  confirm product that user will sale

register item
- save new item for sale

get detail of item
- get detail information of item that user sales

get option and address
- get option and address of item in transaction
